src/main/java/codechicken/nei/recipe/StackInfo.java

```diff
@@ -34,26 +46,31 @@ public static ItemStack loadFromNBT(NBTTagCompound nbtTag)
         return stack;
     }
 
-    public static ItemStack normalize(ItemStack item) {
```

Comment:
this one's fine, but only because you recently added it, and I doubt anyone has taken a dependency on it yet.